### DISCUSSION TITLE
Support für XML-API Addon 2.3 vorhanden

### DISCUSSION THREAD

**Discussion by luftdieb** [SukramJ/aiohomematic#1301] [Q&A] [answered]
Hallo,
die aktuelle Version im RaspberryMatic für XML-API Addon ist 2.3. Ist wird von einem Breaking Change im Changelog berichtet, weil sich die API Version geändert hat aus  Sicherheitsgründen. (https://github.com/homematic-community/XML-API/releases)

Kann jemand bestätigen, dass es damit noch lauffähig ist ?

Danke
Gruß
Stefan

**Comment by SukramJ** [comment, marked as answer]
Die XML-API wird nicht verwendet.
